### ISSUE TITLE
Warnings about NUMA while starting up the container

### ISSUE THREAD

**Issue by w1ndy** [docker-library/mongo#113]
Hi,

I'm running the mongo image on a NUMA-enabled machine, and I get following warning from logs when MongoDB is starting:

> WARNING: You are running on a NUMA machine.
>           We suggest launching mongod like this to avoid performance problems:
>               numactl --interleave=all mongod [other options]

How can I resolve this? Any help is appreciated, thanks.

**Comment by w1ndy** [comment]
Seccomp is preventing numactl from working correctly, because get_mempolicy and set_mempolicy syscall is restricted in docker containers. Possible workarounds include:

1. Run without seccomp: ``docker run --security-opt seccomp=unconfined mongo``
2. Run with [a custom seccomp profile](https://gist.github.com/w1ndy/4aee49aa3a608c977a858542ed5f1ee5): ``docker run  --security-opt seccomp=./seccomp.json mongo``

Hope it helps other people who ran into the same problem. Maybe we can add it into docs? For reference: https://docs.docker.com/engine/security/seccomp/

**Comment by tianon** [comment]
Indeed, adding this to the documentation would be extremely appreciated.

Our current test already includes something like this (creating a custom seccomp profile on the fly): https://github.com/docker-library/official-images/blob/98d3a4d3cecc8b53e0ea547337eeff53ff58710e/test/tests/mongo-basics/run.sh#L6-L53

**Comment by tianon** [comment]
To improve on this, adding `--cap-add SYS_NICE` to your container arguments is sufficient to allow NUMA to do the right thing.
